plugins/auth-backend/README.md

```diff
@@ -158,6 +158,46 @@ To try out SAML, you can use the mock identity provider:
 
 [How to add an auth provider](https://github.com/backstage/backstage/blob/master/docs/auth/add-auth-provider.md)
 
+## Token issuers
+
+By default, Backstage uses the `TokenFactory` for signing the `Authorization` header. The `TokenFactory` generates a
+new private key for every instance every 30 minutes. Public keys are shared via the database backend.
+
+Alternatively users can provide their own public and private key to sign the `Authorization` header with. This can be
+beneficial in scenarios where the token verification implementation aggressively caches the list of keys, and doesn't
+attempt to fetch new ones even if they encounter an unknown kid. To enable this feature add the following configuration
+to your config file:
+
+```yaml
+auth:
+  issuer:
+    mountedTokenIssuer:
+      publicKeyFile: /path/to/public.key
+      privateKeyFile: /path/to/private.key
+```
+
+The private key should be generated using the ES256 algorithm and stored in the PKCS#8 format. The public key should be
+stored in the SPKI format. You can generate the public/private key pair, using openssl by performing the following
```

Comment:
Feel free to add the word openssl to `.github/vale/Vocab/Backstage/accept.txt` as part of this PR to silence vale.

Reply:
Thanks, should be fixed now.